lib/cli/src/commands/wasmer_create_exe_main.c

```diff
@@ -0,0 +1,89 @@
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "wasmer_wasm.h"
+#include "wasm.h"
+#include "my_wasm.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifdef __cplusplus
+}
+#endif
+
+void print_wasmer_error()
+{
+    int error_len = wasmer_last_error_length();
+    printf("Error len: `%d`\n", error_len);
+    char* error_str = (char*) malloc(error_len);
+    wasmer_last_error_message(error_str, error_len);
+    printf("Error str: `%s`\n", error_str);
+}
+
+int main() {
+        printf("Initializing...\n");
+        wasm_config_t* config = wasm_config_new();
+        wasm_config_set_engine(config, OBJECT_FILE);
+        wasm_engine_t* engine = wasm_engine_new_with_config(config);
+        wasm_store_t* store = wasm_store_new(engine);
+
+        wasm_module_t* module = wasmer_object_file_engine_new(store, "qjs.wasm");
+        if (! module) {
+                printf("Failed to create module\n");
+                print_wasmer_error();
+                return -1;
+        }
+        
+        // We have now finished the memory buffer book keeping and we have a valid Module.
+
+        // In this example we're passing some JavaScript source code as a command line argument
+        // to a WASI module that can evaluate JavaScript.
+        wasi_config_t* wasi_config = wasi_config_new("constant_value_here");
+        const char* js_string = "function greet(name) { return JSON.stringify('Hello, ' + name); }; print(greet('World'));";
+        wasi_config_arg(wasi_config, "--eval");
+        wasi_config_arg(wasi_config, js_string);
```

Comment:
We probably don't want this.

We would probably want to replace `constant_value_here` with the first argument to main.
We also would like to relay the rest of the arguments also to WASI.

It would be a good idea to get some other thoughts on mounted dirs and environments. Perhaps those can be provided as configuration to the `create-exe` subcommand.